CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.2.0] - 2022-01-24
+### Changed
+- Added telemetry feature.
+- Improved unit tests coverage
+- Refactored ODD code
```

Comment:
Instead of ODD, let's spell out On-Device Decisioning since this will be read by external audiences that might not know the acryonym.